resources/models/resource.py

```diff
@@ -187,6 +187,9 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
                                                     null=True, blank=True)
     reservable = models.BooleanField(verbose_name=_('Reservable'), default=False)
     reservation_info = models.TextField(verbose_name=_('Reservation info'), null=True, blank=True)
```

Comment:
We talked about this with @petterill, and figured out the difference between this field and the new one. Please add a `help_text` attribute to `reservation_info` with the contents "If there are any special considerations for making reservations, write them here".

Reply:
This change done in 8affc1d.